### DISCUSSION TITLE
Ctrl-Zoom Crash

### DISCUSSION THREAD

**Discussion by meteorquake** [sumatrapdfreader/sumatrapdf#4134] [Ideas]
On this file -
https://archive.org/download/antalyaplants/juncaceae.pdf
If I ctrl-mousewheel zoom over the contents and other places it regularly crashes Sumatra.
3.6.15987 64-bit Win10-64
Cheers,
David

**Comment by meteorquake** [comment]
Just to add I find quite a few files crash in this way...

**Reply by GitHubRulesOK** [reply]
when you get a crash it is best to follow the guidance 
first before the crash is repeated select debug download symbols and ensure it says downloaded (for me my current test version cannot do that as now out of date)
This will ensure only a recent set of debug files is used with the next crash
Induce a repeat of the same crash and submit the 2 small files (not the .dbg ones but the dump **sumatrapdfcrash.dmp**  and the associated **crashreport.txt** in a zip attached to a **new open issue**

**Comment by kjk** [comment]
Was fixes by https://github.com/sumatrapdfreader/sumatrapdf/commit/3445298bb86add02d74fd19622bdab52af5cb53c

**Comment by gcafepgdi** [comment]
> Was fixes by [3445298](https://github.com/sumatrapdfreader/sumatrapdf/commit/3445298bb86add02d74fd19622bdab52af5cb53c)

Fixed the crash when Zoom-In.

Still crashes when Zoom-Out, especially if the document contains some graphics element (picture, art-work).

**Reply by GitHubRulesOK** [reply]
@gcafepgdi 
when you get a crash it is best to follow the guidance
first before the crash is repeated select debug download symbols and ensure it says downloaded (for me my current test version cannot do that as now out of date)
This will ensure only a recent set of debug files is used with the next crash
Induce a repeat of the same crash and submit the 2 small files (not the .dbg ones but the dump sumatrapdfcrash.dmp and the associated crashreport.txt in a zip attached to a new open issue

**Comment by kjk** [comment]
I can't reproduce a crash with the pdf here.

For crashes please open a new issue and follow the guidelines: https://www.sumatrapdfreader.org/docs/Submit-crash-report
